src/lib/party.ts

```diff
@@ -782,3 +801,78 @@ export class Party<P> {
     return party;
   }
 }
+
+function toParty(party: Party): Party_ {
+  let predicate: Party_['predicate'];
+  if (party.predicate === undefined) {
+    predicate = { type: 'accept' };
+  } else if (party.predicate instanceof UInt32) {
+    predicate = { type: 'nonce', value: party.predicate };
+  } else {
+    predicate = { type: 'full', value: party.predicate };
+  }
+  return {
+    predicate,
+    body: {
+      ...party.body,
+      events: party.body.events.events,
+      depth: parseInt(party.body.depth.toString(), 10),
```

Comment:
Would `events` and `body` ever be null? Wondering if it would be good to check for these values if it makes sense.

Reply:
No, they shouldn't be null! I'd say let's rely on the type system and check null/undefined only for values whose type includes null/undefined